shopinvader/components/partner_validator.py

```diff
@@ -4,7 +4,11 @@
 
 from odoo.addons.component.core import Component
 
-from ..exceptions import PartnerNotValidatedError
+# TODO: this exception is not handled yet on Locomotive side.
```

Comment:
@simahawk Do you have tried to raise a ValidatonError?

Reply:
@lmignon nope, but at this stage you'll get not a proper handling of the use case on the website.
I prefer to rely on the use of the flag to display meaningful info for the current user.
Also because this might be optional w/ another backend policy: do you want to block log in or do you want to block purchases only?

Reply:
I prefer to block purchases only...

Reply:
let's keep it like this for now, then we discuss together, maybe w/ @sebastienbeau as well, and we decide what to do. Right no is harmless and it leaves control to the theme.